projects/ui/src/Styles/_topnav.scss

```diff
@@ -151,3 +137,29 @@
     cursor: default;
   }
 }
+
+// Interaction
+.navContent .userLoginArea.loggedIn,
```

Comment:
This feels like a bit of a complex list.  Appreciate that it may be necessary, but perhaps we can throw on a descriptive sentence to make it easier to digest.